### DISCUSSION TITLE
Request - LURO 5M and 1D choose button

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#51] [Bugs and Reports]
Can we make 2 buttons for choosing the game ? Because now it look we have only one game, So if we can make 5M and 1D next to each other? or make some swap button. now it look like hide and seek.

<img width="662" alt="Snímek obrazovky 2024-10-04 v 0 24 18" src="https://github.com/user-attachments/assets/ba464630-5959-4942-b291-b8c757e893f4">

**Comment by bf-monster** [comment]
Hey, sure. @machinedramon What do you think will be the best way to display switcher between games intervals?

**Reply by machinedramon** [reply]
I missed that notification. I’d like a bit more explanation. The idea is to make it clear to the user that these are two separate games, not just a 'mode' change within the same game. What kind of impression or atmosphere should we convey to the user?

**Reply by bf-gabagool** [reply]
Hey @machinedramon 
We want it to be shown on the surface of the page, i. e. you wouldn't need to click somewhere to access the interval switcher. User should see it on LuRo page without any additional clicks.
Please suggest a design solution according to given instructions

**Reply by machinedramon** [reply]
@bf-gabagool  Some time ago, we needed to review the design of the header bar, which is located below the navigation bar. It houses the game icon and other information (today, we have two different bars, one for dynamic games and one for conservative games). During this change, I identified the need for this change and suggested simply including a switch icon next to the game name (which is always accompanied by its specific icon). By default, the page would enter 5M (or 1D) mode and when clicked, it would automatically switch to the new game, without needing to open a modal to select, already switching between modes. It is also possible to place just one select in the same position I suggested, and by default it will also show a game and we can quickly switch between the two options directly from the header without a modal as well. In my opinion, the second option is the best, what do you think?

**Comment by bf-monster** [comment]
Closing as done
